ebi-deployment/README.md

```diff
@@ -0,0 +1,51 @@
+# EBI-deployment
+
+Helm chart to deploy EGA-DATA-API to any Kubernetes cluster.
+
+## Table of Contents
+
+- [Prerequisites](#Prerequisites)
+- [Installing the Chart](#Installing-the-Chart)
+- [Uninstalling the Chart](#Uninstalling-the-Chart)
+
+## Prerequisites
+
+Before deploying the Helm charts `Helm` should be installed. Please follow [these ](https://docs.helm.sh/using_helm#install-helm) instructions to install `Helm`.
+
+Then you should also have a running Kubernetes cluster and configured correctly. You may follow [these](https://github.com/NBISweden/LocalEGA-deploy-SUNET) instructions to create a Kubernetes cluster using the tool KubeNow. It is recommended to create a node with preferably 16GB of RAM so that this Helm Chart can be deployed smoothly.
```

Comment:
This needs to be removed as it is not a public repository.

Reply:
If you want the users to use KubeNow, it's better to refer to the correct documentation (https://kubenow.readthedocs.io/en/stable/)

Reply:
I've changed the instruction url to https://kubenow.readthedocs.io/en/stable/getting_started/bootstrap.html#deploy-on-openstack